extensions/resteasy-classic/resteasy-client/runtime/src/main/java/io/quarkus/restclient/runtime/QuarkusProxyInvocationHandler.java

```diff
@@ -1,12 +1,10 @@
 package io.quarkus.restclient.runtime;
 
+import java.io.Closeable;
 import java.lang.annotation.Annotation;
-import java.lang.reflect.InvocationHandler;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.lang.reflect.Modifier;
-import java.lang.reflect.ParameterizedType;
-import java.lang.reflect.Type;
+import java.lang.reflect.*;
```

Comment:
Please avoid star imports (they are usually added automatically by the IDE, but it can be configured to not do so)